index.js

```diff
@@ -1,12 +1,12 @@
 const express = require('express')
 const spaceX = require('./spaceX')
-const {getEngineers,getTeam, addengineer} = require('./controller/engineers')
+const {getSpaceX,getTeam, addengineer} = require('./controller/engineers')
 const path = require('path')
 const app = express()
 app.use(express.static('public'))
 app.set('view engine', 'pug')
 app.get('/theteam',getTeam)
```

Comment:
this looks good, don't forget spacing for readability.  You did forget one thing though so it never brought me to your main page which is fantastic by the way BUT...
app.get('/', (request, response) => {
    return response.render('index')
  })

you forgot that and therefore your page never came up.  I added that and all went correctly.